content/docs/building-microshift/index.md

```diff
@@ -0,0 +1,203 @@
+---
+title: 'Local Development'
```

Comment:
Need to move this file properly, so the git authorship is maintained.

Reply:
(this is copied from CONTRIBUTING.md)

Reply:
we can git checkout the previous commits to get contributing and then move it to the folder